package.json

```
@@ -28,6 +28,6 @@
     "jscodeshift": "^0.7.0"
   },
   "engines": {
-    "node": ">=6"
+    "node": ">=10"
```

```suggestion
    "node": "10.* || 12.* || >= 14.*"
```

see https://github.com/simplabs/ember-promise-modals/pull/139 😉